### PR TITLE
updated and extended snippets

### PR DESCRIPTION
**Pull request by DataBeast-IT**
Hi!

I've updated the snippets file to reflect Vyper 0.3.2. Current stablevyper version is 0.3.1 but I've added some 0.3.2 things as well, Mostly the DynArray syntax.

### CODE REVIEW

File: snippets/vyper.json [hunk 1]
```diff
@@ -1,85 +1,236 @@
 {
-	".source.solidity": {
-
-        "mapping declaration":{
+    ".source.solidity": {
```
Comment (tintinweb) [inline snippets/vyper.json:2]: huh, I guess I forgot to change this when I copied it from my solidity extension 😅. I think it should be safe to change this to `.source.vyper` 🙏

Comment (DataBeast-IT) [reply snippets/vyper.json:2]: Would you like me to do that and update / resubmit the PR?

Comment (tintinweb) [reply snippets/vyper.json]: that'd be great 🙌

Comment (DataBeast-IT) [reply snippets/vyper.json]: I've added a commit with that change. I see it in the commits tab here so I think it's part of the PR now?

**Comment by tintinweb** [comment]
thank you @DataBeast-IT 🙌

**Comment by Doc-Pixel** [comment]
> 

Because you are on a vyper version lower than 0.3.2. 
Also the vs code extension does not compile the smart contract. It calls vyper, which is a different project all together 😎

If you update Vyper and set the version to at least 0.3.2 I am confident it will work

**Pull request merged**